### ISSUE TITLE
GADRequest.request' is undefined [iOS] -  error from previous GL command [Android]

### ISSUE THREAD

**Issue by Universi** [EddyVerbruggen/nativescript-admob#64]
Hi!

Thank you for the plugin, but it doesn't work for me on iOS and Android.

- Pod updated on iOS -> Ok
- TNS --version -> 5.3.4
- meta-data android added to AndroidManifest with the right app ID and at the right spot.
- iosBannerId, androidBannerId and iosTestDeviceIds set.

I get this error on iOS :
CONSOLE LOG file:///app/tns_modules/nativescript-admob/admob.js:145:18: Error in admob.createBanner: TypeError: GADRequest.request is not a function. (In 'GADRequest.request()', 'GADRequest.request' is undefined)

And this one on Android :

JS: admob createBanner done
JS: displayedEvent : [object Object]
JS: displayedEvent : [object Object]
chromium: [INFO:library_loader_hooks.cc(143)] Chromium logging enabled: level = 0, default verbosity = 0
chromium: [ERROR:gles2_cmd_decoder.cc(2167)] [GroupMarkerNotSet(crbug.com/242999)!:547D98CA]GL ERROR :GL_INVALID_OPERATION : BackFramebuffer::Create: <- error from previous GL command

What can I do, please ?
I thank you in advance

**Comment by EddyVerbruggen** [comment]
Can you remove the platforms folder and try again? And what's the build command you're using?

**Comment by Universi** [comment]
I already removed iOS and Android platform.
Build command is : tns run iOS [or] Android

**Comment by EddyVerbruggen** [comment]
Do you have the same issues with the demo app in this repo?

```shell
git clone https://github.com/EddyVerbruggen/nativescript-admob
cd nativescript-admob/src
npm i
npm run demo.ios
```

**Comment by Universi** [comment]
Your demo works perfectly.

**Comment by EddyVerbruggen** [comment]
Are you able to narrow down the differences between our apps, or do you want me to take a look at yours?

**Comment by Universi** [comment]
I'll try to find the differences between the two, and if I don't find, I'll come back to you in the day.
Thank you so much.

**Comment by Universi** [comment]
I just received an email from Admob telling me that I can't get ads because my payment informations are not filled yet. Could this be the reason for the errors?
A delay of 24 is required for the information entered to be taken into account.
I'm waiting a bit to see if that solves the problem and I keep you informed. Thanks.

**Comment by AlexisWalravens** [comment]
I have the exact same issue, did you find any solution ?

**Comment by bowlesns** [comment]
@EddyVerbruggen I got this error a few months ago but just downgraded tns to temporarily get around it. Got it in the demo app after rebuilding node_modules and platform with tns 5.3.0:
![image](https://user-images.githubusercontent.com/15946490/61255756-ef386080-a737-11e9-9154-a766c4b0c24f.png)

I did some more testing today and it looks like it happens in 5.3.0 forward.  I tried 5.2.2 and it worked fine, and I was running on 5.1.0 previously in my personal app.  I'm happy to do some more investigation or help out anyway I can.

**Comment by bowlesns** [comment]
@AlexisWalravens @Universi Try to downgrade tns to a version below 5.3.0 it may work assuming you did a rebuild after removing platforms folder.  I had the same issue and was able to get ads to appear after that.

**Comment by AlexisWalravens** [comment]
This is not an ideal solution especially that NS 6.0 is out now...

**Comment by EddyVerbruggen** [comment]
I'll take a look.

**Comment by EddyVerbruggen** [comment]
I've just published version 4.0.0 of this plugin with one major change on iOS: the AdMob SDK now requires a change to your `Info.plist`. See [this bit in the README](https://github.com/EddyVerbruggen/nativescript-admob#ios) for details.

While not directly related to this issue it may solve the problem for you, because I was able to run the demo app in this repo with NativeScript 5.4 without issues.
